authors/apps/authentication/tests/test_serializers.py

```diff
@@ -0,0 +1,138 @@
+from django.test import TestCase
+from rest_framework.serializers import ValidationError
+
+from authors.apps.authentication.models import User
+from authors.apps.authentication.serializers import LoginSerializer
+from authors.apps.authentication.serializers import RegistrationSerializer
+from authors.apps.authentication.serializers import UserSerializer
+
+
+class RegistrationSerializerTests(TestCase):
+    """
+    Unit tests for the RegistrationSerializer
+    class defined in our serializers.
+    """
+
+    def setUp(self):
+        self.user = {
+            "username": "user",
+            "email": "user@mail.com",
+            "password": "password"
+        }
+
+    def test_register_method(self):
+        """ Test whetther the create method works. """
+        serializer = RegistrationSerializer(data=self.user)
+        self.assertTrue(serializer.is_valid())
+        saved_user = serializer.save()
+        self.assertIsInstance(saved_user, User)
+
+
+class LoginSerializerTests(TestCase):
+    """
+    Unit tests for the LoginSerializer class
+    defined in our serializers.
+    """
+
+    def test_login_no_password_validation(self):
+        """ Test whether users can log in without a password. """
+        data = {
```

Comment:
I would suggest adding a 'user' property or 'data' property to the test class so as to reduce the changes it would take to when adjusting the user or 'data' parameters.